vm-virtio/src/console.rs

```diff
@@ -372,6 +372,7 @@ impl VirtioConsoleConfig {
 
 /// Virtio device for exposing console to the guest OS through virtio.
 pub struct Console {
+    id: String,
```

Comment:
Ditto.

Reply:
Oh no this is different here, I want to keep it this way because the virtio console device might be reused by some other VMMs, and they wouldn't want the console id to be hardcoded.
It's on purpose that I set the identifiers from the DeviceManager because it controls the IDs and it can ensure they're unique.